Formula/hashcat.rb

```diff
@@ -0,0 +1,21 @@
+class Hashcat < Formula
+  desc "World's fastest and most advanced password recovery utility"
+  homepage "https://hashcat.net/hashcat/"
+  url "https://hashcat.net/files/hashcat-#{version}.tar.gz"
+  version "3.10"
+  sha256 "3b555e5f7b35ab6a4558bc460f28d80b32f5a211bf9e08d6a1ba1bad5203e3e9"
+
+  depends_on "gnu-sed" => :build
+
+  def install
+    args = %W[
+      CC=#{ENV.cc}
+      PREFIX=#{prefix}
+    ]
+    system "make", "install", *args
+  end
+
+  test do
+    system "hashcat", "--version"
```

Comment:
Could you use `assert_match ..., shell_output(...)` to do a basic check on the output here?